gpt_lw/loss.py

```diff
@@ -50,7 +50,9 @@ def tensor_abspos(x):
     def weighted_nt(variables, key, xt, xtp1):
         logits, state = forward(model, variables, key, xt)
         token_loss = optax.losses.softmax_cross_entropy_with_integer_labels(logits, xtp1)
+
         weights = weight_fn(xt)
+        weights *= model.config.seq_len / weights.sum(axis=1, keepdims=True)
```

Comment:
maybe xt.shape[1] instead of model.config.seq_len? I'm not sure how long we should keep seq_len in model configs